docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc

```diff
@@ -0,0 +1,42 @@
+= Manually Restore DBaaS Backup
+
+== icon:glasses[] Overview
+
+NOTE: You can't restore from a `+hobbyist-2+` instance. You have to scale it up first.
+
+This documentation is for manually restoring a DbaaS instance.
+
+== Scale up Instance
+
+[source,shell]
+----
+exo dbaas update -z ch-gva-2 $myinstance --plan startup-4
+----
+
+== Restore Instance
+
+[source,shell]
+----
+# List available Backups
+exo dbaas show $myinstance -z ch-gva-2 --backups
+
+┼─────────────────────────────────────┼───────────────────────────────┼──────────┼
+│                NAME                 │             DATE              │   SIZE   │
+┼─────────────────────────────────────┼───────────────────────────────┼──────────┼
+│ 2021-10-30_20-28_0.00000000.pghoard │ 2021-10-30 20:28:01 +0000 UTC │ 36341760 │
+│ 2021-10-31_20-28_0.00000000.pghoard │ 2021-10-31 20:28:04 +0000 UTC │ 36382720 │
+┼─────────────────────────────────────┼───────────────────────────────┼──────────┼
+
+# Create a fork from backup
+# With point-in-time
+exo dbaas create pg startup-4 fork-test-pg --pg-recovery-backup-time 2021-10-30 23:35 --pg-fork-from $myinstance -z ch-gva-2
+
+# Without point-in-time (restores latest available)
+exo dbaas create pg startup-4 fork-test-pg --pg-fork-from $myinstance -z ch-gva-2
+----
+
+There will be a new instance, which is not AppCat managed!
```

Comment:
Crossplane should be able to adopt an instance by setting the `externalName` (if I remember correctly). That would be much easier than doing another dump/restore dance.

Reply:
Good point, but haven't tried that yet...

Reply:
Maybe something to plan for? Would make restoring much easier IMHO...

Reply:
I agree, but for the time being, if shit hits the fan we need to do the dump dance to do restores.

We can rework this page again once we have implemented either the `external-name` or the ability to do the restore via claim.

Reply:
Created a tech-debt: https://github.com/vshn/provider-exoscale/issues/78